src/main/java/com/strandls/utility/service/impl/UtilityServiceImpl.java

```diff
@@ -608,4 +607,48 @@ public HomePageData reorderHomePageSlider(HttpServletRequest request, List<Reord
 		return null;
 	}
 
+	@Override
+	public List<Long> getResourceIds(String phrase, String type, String tagRefId) {
+		List<Long> resourceIds = new ArrayList<>();
+		List<String> phraseList = Arrays.asList(phrase.split(","));
+		List<String> typeList = Arrays.asList(type.split(","));
+		List<String> tagRefIdList = Arrays.asList(tagRefId.split(","));
+
+		for (String item : phraseList) {
+			Tags tag = tagsDao.fetchTag(item);
+			if (tag != null) {
+				List<TagLinks> taglinks;
+				if (!typeList.isEmpty() && !typeList.contains("all")) {
+					for (String typeItem : typeList) {
+						taglinks = tagLinkDao.findResourceTags(typeItem, tag.getId(), null);
+						resourceIds.addAll(getTagReferList(taglinks));
```

Comment:
I think its not possible to do it because  we have to fetch resource ids based on multiple tags , for each tag we have to use two tables , the first one is Tag table , based on the user phrase input i have to get the tag and its id , and form TagLinks table form the fetched tag id i have to fetch the resource id , there will be multiple resource id on a single tag , thus i am adding it in a list . we have to discuss this further .